values_examples/calico-or-kuberouter-NodePort-extIP.yaml

```diff
@@ -0,0 +1,61 @@
+# Default values for vkaci.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+# VKACI APIC Variables
+# All of these will need to be set correctly
+apicIps: "10.67.185.102,10.67.185.41,10.67.185.42"
+apicCertName: "nkt_user_Xvpp3k"
+apicUsername: "nkt_user_Xvpp3k"
+vrfTenant: "Cilium"
+vrfName: "Cilium"
+# Get the APIC key data by encoding your key file as a base64 string
+# NOTE: It would have been nice to be able to pass in a filename instead so Helm can do it for us
+# but I found no way for Helm to access user files outside of the chart.
+apicKeyData: "LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlDWEFJQkFBS0JnUURKV3N1Mi9OOEJDb0ZkWDFDOUh3cW15QkFJcUE3TExoWEVnQjNXcmM3S245aGxKSHNyCklxMkdxWExWRTNQRkY5ZUZQcU4ybzRMbnlHWkRTNDYreEVvTWM3M2YvTDU0eDBDRTZoT1N0cW1mYmFEOE5ybVYKK21XcUFUeXlrU0o0anpVRWp6bVYyRzRyWmRNbXBvek9VNER0OUE4ZWtHLzRVZ3d4NnZ2aXUwNHhaUUlEQVFBQgpBb0dBRnNkVXdIQys4YWswbFZHUkdyUUdTblBiakg4bW5VbXJZTDExb3JHcThUVkFEOEVleGZSNFdRR1dZaEIvClNrNjJMUGEzRzFiSUdseUxaUldKSW4rbU9LckxMZnZtNEJLOW80YkJmTS9iZUNmbEFQajZhVHoxcmllck5hOU8KY2MvTUM5aGxRNDJJNUJYaU8zRlNSK1hqTU5HQmhzSVNnZkc4Z3ErUXkyUFRUa0VDUVFENkd0S0h5dlFHVVN2SgpyeEhyRDRZZS9qK21lZEc0MVhjRCtoTFcxei9Rd21uRXZYWUNpSUNWM0JURFM5Y1lGZlNSamtlZm5BRUg1akZDCnZhc2tndmxwQWtFQXpobk9jQ2dTcnFUMmYxQWxVTHl5WE9RSk9CVW1hUGJJRy9HMXE2WWlGWnE5Y1c0YmRRL08KV2N1RVp2VThTT3B1alpWNm41SktiWDJmanVxZmtkM2NuUUpBQWtOVjNsYXV5SlBaY3BTNzg4V0xNQ0thenRYUgpXUHZOZk9LNVl1cEZzSXpoeWg3d3Yyc2V4cnhjRDd4UzhPdlo3NnM2NkszUHNKT2JwSC8wMlhxNTZRSkFkY2cvCnRzN2Y1bGkxMmdvbVdSNk5YQktoeGlBb0pFSXNGL1FwVzJBRkNJMjNCL1RoMGlOblROL0cyY1A2WUwxQTByOEQKSmVCSGtvMEJPMzZRRko5UzdRSkJBTkRFZ01FRG5KZG43cEJNYzNOeFBuNmpNb0RQaUgrNEFKQjlJRXpBbjZrUwpUbnkyNDFhakFoV2VNMTJlUXhMWXd6Z3FkSm02aFMzRkZzUkRnbUR2cmlFPQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQ=="
```

Comment:
As discussed in the previous messages from 12/07 last year I am just emphasizing on this again that let’s not expose secret keys and passwords even if they’re examples specifically in an opensource project. You can have example config files but there’s absolutely no reason for example key data to be added.

Reply:
Keys removed. Anyway are just the one to access my APIC that are deleted every time a new cluster is deployed